docker/Dockerfile

```diff
@@ -86,6 +86,9 @@ COPY --from=jre-build /javaruntime $JAVA_HOME
 COPY scripts /opt/bookkeeper/scripts
 RUN chmod +x -R /opt/bookkeeper/scripts/
 
+RUN chown -R 10000:0 /opt/bookkeeper
```

Comment:
I think it's better to specifically list the directories that are allowed to be writable, [similar to how this is handled in Pulsar's Dockerfile](https://github.com/apache/pulsar/blob/47f204fb20d324d6794e3b57e06807af66b799bc/docker/pulsar/Dockerfile#L38-L48).

Reply:
Do you want to limit some directories to be writable/readonly?

Reply:
Done.